File: data-quality/src/test/java/it/gov/daf/dataquality/analysis/email/checkers/FreeMailCheckerTest.java
package it.gov.daf.dataquality.analysis.email.checkers;

import junit.framework.TestCase;
import org.junit.Test;

/**
 * @author Marco Bianchi (bianchi74@gmail.com)
 */
public class FreeMailCheckerTest extends TestCase {

    @Test
    public void testIsChecked(){
        Checker checker = FreeMailChecker.getInstance();
        assert(! checker.isChecked("someone@agid.gov.it"));
        assert(checker.isChecked("someone@gmail.com"));
    }

}

File: data-quality/trash/FreeMailCheckerTest.scala
package it.gov.daf.dataquality.analysis.email.checkers

import org.scalatest.FunSuite

/**
  * @author Marco Bianchi (bianchi74@gmail.com)
  */
class FreeMailCheckerTest extends FunSuite {

  test("testGetInstance"){
    FreeMailChecker.getInstance()
  }

  test("testIsChecked") {
    val checker = FreeMailChecker.getInstance()
    assert(! checker.isChecked("someone@agid.gov.it"))
    assert(checker.isChecked("someone@gmail.com"))
  }

}
